client/.env.production

```diff
@@ -1,10 +1,10 @@
 GENERATE_SOURCEMAP='false'
-REACT_APP_EDNA_PLATFORM_SERVICE_URL='https://edna-lti-ab-platfomrs.azurewebsites.net/api'
-REACT_APP_EDNA_USERS_SERVICE_URL='https://edna-lti-ab-users.azurewebsites.net/api'
-REACT_APP_EDNA_LEARN_CONTENT='https://edna-lti-ab-learncontent.azurewebsites.net/api'
-REACT_APP_EDNA_LINKS_SERVICE_URL='https://edna-lti-ab-links.azurewebsites.net/api'
-REACT_APP_EDNA_ASSIGNMENT_SERVICE_URL='https://edna-lti-ab-assignments.azurewebsites.net/api'
+REACT_APP_EDNA_PLATFORM_SERVICE_URL='https://platforms-crbr22nms.azurewebsites.net/api'
```

Comment:
Any reason for making these changes?

Reply:
Checked this in by mistake. Reverted the change.